source/include/FreeRTOSIPConfigDefaults.h

```diff
@@ -800,6 +800,7 @@
  * It should be noted that it is most efficient to drop unwanted packets
  * as early as possible.
  */
+/* coverity[misra_c_2012_rule_5_4_violation] */
```

Comment:
Can we add some comment about what this rule is and why are suppressing it?
(Maybe we can copy the PR description).

Doing it, in my opinion, should make the comment more understandable to the reader - otherwise, one unaware about MISRA/coverity might get confused by this.

Reply:
i don't think it is legal to state what the rule is

Reply:
will add a little description on top

Reply:
The problem is that the project docs are not up front about this non-compliance with the C90 / C89 standard. I have yet to run into a compiler that actually only checks the first 31 characters though.

In this case, all macros starting with ipconfigETHERNET_DRIVER_FILTER could be considered the same by a compiler.

Or we could just claim C99 compliance instead of C89/C90.